Gems/AWSGameLift/Code/AWSGameLiftClient/Source/AWSGameLiftClientLocalTicketTracker.cpp

```diff
@@ -0,0 +1,168 @@
+/*
```

Comment:
Should link change to your RFC

Reply:
do we have any policy that we should provide RFC link, it doesn't make sense to me